src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -61,7 +65,16 @@ public static (List<Error> errors, MarkdownDocument ast) Parse(string content, M
 
                     var html = Markdown.ToHtml(markdown, s_markdownPipelines[(int)pipelineType]);
 
-                    return (status.Errors, html);
+                    var htmlWithRelativeLink = HtmlUtility.TransformLinks(html, (href, _) =>
+                    {
+                        if (href.StartsWith(RelativeUrlMarker))
+                        {
+                            return UrlUtility.GetRelativeUrl(file.SiteUrl, href.Substring(RelativeUrlMarker.Length));
+                        }
```

Comment:
will this bring perf concern?

Reply:
not likely, `azure-docs-pr (00:02:28.9939835, 0 more diff)`